src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -113,6 +118,8 @@ private async Task UpdateImportTableRow(Guid releaseId, string dataFileName, int
         private ImportMessage BuildMessage(string dataFileName,
             string metaFileName,
             Guid releaseId,
+            int numBatches,
+            int batchNo,
```

Comment:
Would suggest refactoring these to use default parameters of 0 or 1 (whatever is actually correct)